### ISSUE TITLE
Probabilistic Backpropagation for ScalableLearning of Bayesian Neural Networks

### ISSUE THREAD

**Issue by dlehdgus9887** [dlehdgus9887/paper-log-own#4]
## 어떤 내용의 논문인가요? 👋

간략하게 어떤 내용의 논문인지 작성해 주세요! (짧게 1-2줄 이어도 좋아요!)

## Abstract (요약) 🕵🏻‍♂️

Large  multilayer  neural  networks  trained  withbackpropagation have recently achieved state-of-the-art results in a wide range of problems. How-ever, using backprop for neural net learning stillhas  some  disadvantages,  e.g.,  having  to  tune  alarge  number  of  hyperparameters  to  the  data,lack  of  calibrated  probabilistic  predictions,  anda tendency to overfit the training data.  In prin-ciple,  the Bayesian approach to learning neuralnetworks  does  not  have  these  problems.   How-ever, existing Bayesian techniques lack scalabil-ity  to  large  dataset  and  network  sizes.   In  thiswork  we  present  a  novel  scalable  method  forlearning Bayesian neural networks, called proba-bilistic backpropagation (PBP). Similar to classi-cal  backpropagation,  PBP  works  by  computinga  forward  propagation  of  probabilities  throughthe network and then doing a backward computa-tion of gradients. A series of experiments on tenreal-world datasets show that PBP is significantlyfaster than other techniques, while offering com-petitive predictive abilities. Our experiments alsoshow that PBP provides accurate estimates of theposterior variance on the network weights.

## 이 논문을 읽어서 무엇을 배울 수 있는지 알려주세요! 🤔

이 논문을 제대로 읽었을 때 어떤 지식을 얻을 수 있을까요?

## 같이 읽어보면 좋을 만한 글이나 이슈가 있을까요?

만약에 있다면 자유롭게 작성해 주세요!

## 레퍼런스의 URL을 알려주세요! 🔗

markdown 으로 축약하지 말고, 원본 링크 그대로 그냥 적어주세요!
